framework/include/interfaces/Coupleable.h

```diff
@@ -158,7 +158,7 @@ class Coupleable
    * @see Kernel::_u
    */
   virtual const VariableValue & coupledValue(const std::string & var_name,
-                                             unsigned int comp = 0) const;
+                                             unsigned int comp = libMesh::invalid_uint) const;
```

Comment:
Lots of other functions have the 0 as the default. Do we need to change those?

Reply:
Yeah, that should be investigated. `coupledValue` is the most important one IMO, and I wanted to see how this impacts our tests.